orbital-mechanics/Lecture4/Lecture4.ipynb

```diff
@@ -0,0 +1,397 @@
+{
```

Comment:
<p>I think you haven't uploaded the figure. Please create a directory inside the Lecture4 directory called imgs; put your images in there.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angadhn/SpacecraftDynamics/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>